apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo3/network/http/BatchingHttpEngine.kt

```diff
@@ -55,11 +55,18 @@ import kotlin.jvm.JvmStatic
  *
  * @param batchIntervalMillis the interval between two batches
  * @param maxBatchSize always send the batch when this threshold is reached
+ * @param exposeErrorBody configures whether to expose the error body in [ApolloHttpException].
+ *
+ * If you're setting this to `true`, you **must** catch [ApolloHttpException] and close the body explicitly
+ * to avoid sockets and other resources leaking.
+ *
+ * Default: false
```

Comment:
Nit: I think this needs to be on one line for KDoc to understand we're still on the `@param`
```suggestion
 * @param exposeErrorBody configures whether to expose the error body in [ApolloHttpException]. If you're setting this to `true`, you **must** catch [ApolloHttpException] and close the body explicitly to avoid sockets and other resources leaking. Default: false
```

Reply:
This is what I get with the current KDoc:

![Screenshot 2021-12-09 at 14 38 25](https://user-images.githubusercontent.com/3974977/145406603-d6161799-80d6-4ac5-93c8-34913ba76e31.png)

Looks okay-ish to me?

Reply:
Thank you for checking, it works differently that I thought it would - TIL :) You can ignore my suggestion then!